docker/pythonpath/superset_config.py

```diff
@@ -121,15 +121,49 @@ class CeleryConfig(object):
 # Custom config for biocultural monitoring deployments by CMI
 AUTH0_DOMAIN = get_env_variable("AUTH0_DOMAIN")
 
+# Define translations
+translations = {
+    "Welcome! Please sign up or log in with Auth0 below to access the application.": {
+        "pt_BR": "Bem-vindo! Por favor, inscreva-se ou faça login com Auth0 abaixo para acessar o aplicativo.",
+        "en": "Welcome! Please sign up or log in with Auth0 below to access the application.",
+        "nl": "Welkom! Meld u aan of log in met Auth0 hieronder om toegang te krijgen tot de applicatie.",
+        "es": "¡Bienvenido! Regístrese o inicie sesión con Auth0 a continuación para acceder a la aplicación.",
+        "fr": "Bienvenue! Veuillez vous inscrire ou vous connecter avec Auth0 ci-dessous pour accéder à l'application."
+    },
+    "The request to sign in was denied.": {
+        "pt_BR": "O pedido de login foi negado.",
+        "en": "The request to sign in was denied.",
+        "nl": "Het verzoek om in te loggen werd geweigerd.",
+        "es": "La solicitud de inicio de sesión fue denegada.",
+        "fr": "La demande de connexion a été refusée."
+    },
+    "You are not yet authorized to access this application. Please contact a GuardianConnector administrator for access.": {
+        "pt_BR": "Você ainda não está autorizado a acessar este aplicativo. Por favor, entre em contato com um administrador do GuardianConnector para obter acesso.",
+        "en": "You are not yet authorized to access this application. Please contact a GuardianConnector administrator for access.",
+        "nl": "U bent nog niet gemachtigd om toegang te krijgen tot deze applicatie. Neem contact op met een GuardianConnector-beheerder voor toegang.",
+        "es": "Aún no está autorizado para acceder a esta aplicación. Comuníquese con un administrador de GuardianConnector para obtener acceso.",
+        "fr": "Vous n'êtes pas encore autorisé à accéder à cette application. Veuillez contacter un administrateur de GuardianConnector pour obtenir l'accès."
+    }
+}
+
+def translate(message):
+    locale = str(get_locale())
+    return translations.get(message, {}).get(locale, message)
+
 # Extend the default AuthOAuthView to override the default message when the user is not authorized
 class CustomAuthOAuthView(AuthOAuthView):
+    @expose("/login")
+    def login(self) -> WerkzeugResponse:
+        flash(translate("Welcome! Please sign up or log in with Auth0 below to access the application."), "info")
```

Comment:
On an app I previously worked on, we realized people were confused about "sign in" vs "sign  up" vs "log in" if our language didn't match exactly (by which, here, I mean "match the Auth0 button")

It was actually complicated by using a SSO provider (as we are now) which doesn't let you choose at this stage whether you will create an account vs login with an existing one; there's only one button and that confuses new users who think they don't have an account.  Nothing we can do about that, but my main suggestion is to make sure the term in our flashed message matches the Auth0 button's term.

Reply:
> Welcome! Please sign up or log in by pressing 'Sign in with auth0' to access the application"?

I like this one. :+1: